node/src/main/scala/coop/rchain/node/Main.scala

```diff
@@ -182,6 +183,27 @@ object Main {
     ) >> program
   }
 
+  /**
+    * Returns port for repl/eval mode:
+    *   grpcPort if passed
+    *   defaults.conf/api-server/port-grpc-internal otherwise
+    * @param options command line options
+    * @return port
+    */
+  private def replPort(options: commandline.Options): Int =
+    if (options.grpcPort.isSupplied) {
+      options.grpcPort()
+    } else {
+      final case class Port(portGrpcInternal: Int)
+      Configuration
+        .defaultConf()
+        .at("api-server")
+        .load[Port]
+        .right
+        .getOrElse(Port(options.grpcPort()))
+        .portGrpcInternal
```

Comment:
Can you have the same without intriducing case class?
```scala
Configuration
        .defaultConf()
        .at("api-server. port-grpc-internal")
        .load[Int]
        .right
        .getOrElse(options.grpcPort())
```

Reply:
@nzpr, yes, it works. Thanks for the tip!